rollingpin/args.py

```diff
@@ -1,13 +1,24 @@
 import argparse
+import os
+import sys
 
 
 PAUSEAFTER_DEFAULT = 1
 
 
 class ExtendList(argparse.Action):
+    def __init__(self, *args, **kwargs):
+        super(ExtendList, self).__init__(*args, **kwargs)
+        if isinstance(self.default, list):
+            self.true_default = self.default[:]
+        else:
+            self.true_default = self.default
 
     def __call__(self, parser, namespace, values, option_string):
         list_to_extend = getattr(namespace, self.dest)
+        if self.true_default:  # if a default is defined, remove it before extending
+            for default in self.true_default:
+                list_to_extend.remove(default)
```

Comment:
Also some form of the above explanation as an inline comment would be good for posterity. Thanks for the breakdown!